conntrollers/controller.js

```diff
@@ -0,0 +1,11 @@
+class Controller {
+
+    sendHelloMessage(req, res) {
+        res.json({
+            message: 'Hello my dear friend!'
```

Comment:
Please, don't forget about semicolons at the end of the lines)

Reply:
ok